src/libDirectoryService/DirectoryService.cpp

```diff
@@ -86,7 +86,14 @@ void DirectoryService::StartSynchronization() {
     }
   };
 
+  auto func2 = [this]() -> void {
+    bool result = m_mediator.m_lookup->FetchDSInfoLoop();
+    if (!result) {
+      LOG_GENERAL(WARNING, "Unable to fetch DS info");
+    }
+  };
   DetachedFunction(1, func);
+  DetachedFunction(1, func2);
```

Comment:
FetchDSInfoLoop() will call GetDSInfoFromLookupNodes(), and need offline lookup node as well. Thus, We need to make sure FetchDSInfoLoop() be called after GetOfflineLookups() finished, just like FetchLatestTxBlocks().

Reply:
if you see the function start sync it is there.